### PR TITLE
Prep for release 2.19.0

### PR DESCRIPTION
**Pull request by makarandp0**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### Description

prepares adaptive simulcast for the release.

### Before review
* [x] Updated CHANGELOG.md if necessary
* [x] Added unit tests if necessary
* [ ] Updated affected documentation
* [ ] Verified locally with `npm test`
* [ ] Manually sanity tested running locally
* [ ] Included screenshot as PR comment (if needed)
* [x] Ready for review

### CODE REVIEW

**Review by manjeshbhargav — APPROVED** [review]

**Review by charliesantos — APPROVED** [review]
🎉

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,47 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Version 1.x reached End of Life on September 8th, 2021.** See the changelog entry [here](https://www.twilio.com/changelog/end-of-life-complete-for-unsupported-versions-of-the-programmable-video-sdk). Support for the 1.x version ended on December 4th, 2020.
 
+2.19.0 (January 31, 2022)
+=========================
+
+New Features
+------------
+
+- This release introduces a new beta feature **Adaptive Simulcast**. This opt-in feature can be enabled by setting `preferredVideoCodecs="auto"` in ConnectOptions. When joining a group room with this feature enabled, the SDK will use VP8 simulcast, and will enable/disable simulcast layers dynamically, thus improving bandwidth and CPU usage. It works best when used along with `Client Track Switch Off Control` and `Video Content Preferences`. These two flags allow the SFU to determine which simulcast layers are needed, thus allowing it to disable the layers not needed on publisher side. The beta currently does not support setting a max bitrate for your simulcast layers.
```
Comment (anthonylazaro) [inline CHANGELOG.md:11]: This feature is not beta

Comment (anthonylazaro) [reply CHANGELOG.md:11]: "thus improving bandwidth and CPU usage for the publishing client"

"this feature cannot be used alongside maxVideoBitrate"

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,47 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
...
+
+Known Limitations
+-----------------
+
+- Specifying `preferredVideoCodecs="auto"` will revert to unicast in the following cases:
+  - The publisher is using Firefox.
+  - The publisher has preferred the H264 codec.
+  - The Room is configured to support only the H264 codec.
+  - Peer-to-Peer Rooms
+- When the Room is being recorded, the SFU will not disable any simulcast layers of the publisher's VideoTrack.
```
Comment (anthonylazaro) [inline CHANGELOG.md:39]: When the "participant" is being recorded

**Review by anthonylazaro — CHANGES_REQUESTED** [review]
Made some comments for changes

File: CHANGELOG.md (hunk 2 above)
Comment (makarandp0) [reply CHANGELOG.md:39]: @nicopernas, can you confirm this part. Does SFU disable Adaptive Simulcast when the "participant" is being recorded or when  "room" is being recorded?

**Comment by makarandp0** [comment]
@anthonylazaro, Thank you for your suggestions. I have included them in https://github.com/twilio/twilio-video.js/pull/1686/commits/8e610216d0ff5b623d894cc4ffa1df685a0d4e55

**Review by anthonylazaro — APPROVED** [review]

**Comment by makarandp0** [comment]
I will merge this PR into master now - to open develop for next changes. We will cut the release on Monday as planned.
If anybody has more feedback on change log, I will incorporate it in master.

**Pull request merged**